app/konnect/configure/runtime-manager/runtime-groups/manage-entities.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Manage Entities in a Runtime Group
+no_version: true
+---
+
+You can manage any {{site.base_gateway}} Services, Routes, consumer-scoped
+entities, and global entities from within a runtime group.
+
+A **global** entity is a set of configurations that apply to, or can be used
+by, all objects in a runtime group. For example, if you set up a Proxy Caching
+plugin in the default runtime group and set it to `Global`,
+the plugin configuration will apply to all Services in the group.
+
+Consumers, SNIs, upstreams, and certificates are all global entities. Plugins
+can either be global or scoped.
+
+## Configurable Gateway entities in Konnect
+
+Through a runtime group, you can manage the following entities:
+
+### Gateway Services
+
+The **Gateway Service** configuration page lists all {{site.base_gateway}} Services
+in the runtime group. Service entities are abstractions of each of your own
+upstream services, such as a data transformation microservice, or a billing API.
+
+Gateway Services can be exposed in ServiceHub, or managed though Runtime Manager
```

Comment:
@haydenlam I thought we agreed to talk about ServiceHub only in the docs, and allow customers that need the RG config method to use it via deck with their account manager?

Reply:
Afaik, the approach we decided on was to leave RG config out of the getting started guides. I was asked to still write up reference docs for it, though. Is this still too much?